otx/api/entities/label_schema.py

```diff
@@ -558,6 +558,68 @@ def from_labels(cls, labels: Sequence[LabelEntity]) -> "LabelSchemaEntity":
         label_group = LabelGroup(name="from_label_list", labels=labels)
         return LabelSchemaEntity(label_groups=[label_group])
 
+    def resolve_labels_greedy(self, scored_labels: List[ScoredLabel]) -> List[ScoredLabel]:
```

Comment:
```suggestion
    def resolve_labels_greedily(self, scored_labels: List[ScoredLabel]) -> List[ScoredLabel]:
```
You need an adverb, not adjective